<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1323</width>
    <height>659</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ATM柜台机</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="btn_Help">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>460</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image: url(:/images/img/HelpBackgroud.jpg);
border:none;color:
rgb(255, 255, 255);
}
QPushButton:hover{
border-image: url(:/images/img/HelpBackground3.jpg);
border:none;
color:rgb(255, 255, 255);
}
QPushButton:checked{
border-image: url(:/images/img/HelpBackground2.jpg);
border:none;
color:rgb(255, 255, 255)
}</string>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>40</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>中国工商银行</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中国农业银行</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中国建设银行</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>交通银行</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中国银行</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>40</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>选择银行:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_BankNum">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>90</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>22</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLineEdit
{
color: rgb(255, 255, 255);
background-image: url(:/images/img/blueBackgroud2.jpg);
}


QLineEdit:hover
{
border:2px solid #34495E;
color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>19</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>90</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>银行卡号:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_BankPwd">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>140</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/img/blueBackgroud2.jpg);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>140</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>卡号密码:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>179</y>
     <width>391</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QGroupBox {
    /* 背景渐变色*/
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #E0E0E0, stop: 1 #FFFFFF);

    /* 边框 */
    border: 2px solid black;

    /* 倒角 */
    border-radius: 5px;

    /* 就像墙上挂着的两个相框,margin指的是相框与相框的距离
       padding指的是每个相框里照片与相框边框的距离 */
    margin-top: 20px;
}

/* 标题设置 */
QGroupBox::title {
    /* 位置 */
    subcontrol-origin: margin;
    subcontrol-position: top center;

    /* 内边框,上下和左右 */
    padding: 10px 15px;

    /* 颜色 */
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF0ECE, stop: 1 #FFFFFF);
}

QGroupBox::indicator {
    width: 13px;
    height: 13px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请输入密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Please Enter your PIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/img/blueBackgroud2.jpg);
color: rgb(255, 255, 255);


/*QPushButton{
border-image: url(:/images/img/blueBackgroud2.jpg) 0 0 0 0;
border:none;color:
rgb(255, 255, 255);
}
QPushButton:hover{
border-image: url(:/images/img/blueBackgroud3.jpg);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
border:none;
color:rgb(255, 255, 255);
}
QPushButton:checked{
border-image: url(:/images/img/blueBackgroud2.jpg) 0 0 0 0;
border:none;
color:rgb(255, 255, 255)
}*/
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-image: url(:/images/img/blueBackgroud.jpg);
color: rgb(255, 255, 255);*/

background-image: url(:/images/img/blueBackgroud2.jpg);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_9">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-image: url(:/images/img/blueBackgroud.jpg);
color: rgb(255, 255, 255);*/

background-image: url(:/images/img/blueBackgroud2.jpg);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-image: url(:/images/img/blueBackgroud.jpg);
color: rgb(255, 255, 255);*/

background-image: url(:/images/img/blueBackgroud2.jpg);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-image: url(:/images/img/blueBackgroud.jpg);
color: rgb(255, 255, 255);*/

background-image: url(:/images/img/blueBackgroud2.jpg);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-image: url(:/images/img/blueBackgroud.jpg);
color: rgb(255, 255, 255);*/

background-image: url(:/images/img/blueBackgroud2.jpg);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-image: url(:/images/img/blueBackgroud.jpg);
color: rgb(255, 255, 255);*/

background-image: url(:/images/img/blueBackgroud2.jpg);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-image: url(:/images/img/blueBackgroud.jpg);
color: rgb(255, 255, 255);*/

background-image: url(:/images/img/blueBackgroud2.jpg);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>260</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-image: url(:/images/img/blueBackgroud.jpg);
color: rgb(255, 255, 255);*/

background-image: url(:/images/img/blueBackgroud2.jpg);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_xh">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-image: url(:/images/img/blueBackgroud.jpg);
color: rgb(255, 255, 255);*/

background-image: url(:/images/img/blueBackgroud2.jpg);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_jh">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>330</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-image: url(:/images/img/blueBackgroud.jpg);
color: rgb(255, 255, 255);*/

background-image: url(:/images/img/blueBackgroud2.jpg);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>#</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_0">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-image: url(:/images/img/blueBackgroud.jpg);
color: rgb(255, 255, 255);*/

background-image: url(:/images/img/blueBackgroud2.jpg);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_OK">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>200</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/img/blueBackgroud3.jpg);
color: rgb(255, 255, 255);


/*QPushButton{

}
QPushButton:hover{
background-image: url(:/images/img/blueBackgroud2.jpg);
border:none;
color:rgb(255, 255, 255);
}
QPushButton:checked{
background-image: url(:/images/img/blueBackgroud3.jpg);
border:none;
color:rgb(255, 255, 255)
}*/</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Back">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>290</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/img/blueBackgroud3.jpg);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>380</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/img/blueBackgroud3.jpg);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_refundcards">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>540</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/img/redBackgroud.jpg);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>620</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>系统服务热线:18678501532</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Bankstate">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>620</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>620</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>led_BankNum</tabstop>
  <tabstop>led_BankPwd</tabstop>
  <tabstop>btn_OK</tabstop>
  <tabstop>btn_Back</tabstop>
  <tabstop>btn_clear</tabstop>
  <tabstop>btn_refundcards</tabstop>
  <tabstop>Btn_4</tabstop>
  <tabstop>btn_Help</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>Btn_8</tabstop>
  <tabstop>Btn_9</tabstop>
  <tabstop>Btn_6</tabstop>
  <tabstop>Btn_5</tabstop>
  <tabstop>Btn_1</tabstop>
  <tabstop>Btn_2</tabstop>
  <tabstop>Btn_3</tabstop>
  <tabstop>Btn_xh</tabstop>
  <tabstop>Btn_jh</tabstop>
  <tabstop>Btn_0</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
